libs/common/src/constants/date-iana-timezones.ts

```diff
@@ -0,0 +1,8 @@
+import { DateIanaTimezone } from '../types';
+
+/**
+ * IANA timezones used in the codebase
+ * Note - If needing a full list, we should look to Intl.supportedValuesOf('timeZone')
+ * {@link https://www.iana.org/time-zones}
+ */
+export const DATE_IANA_TIMEZONES = { LONDON: 'Europe/London' } as const satisfies Record<string, DateIanaTimezone>;
```

Comment:
Can this please be picked from the env variable `TZ`.

Reply:
I'm slightly cautious of this -- (prior to this pr) our code currently references everywhere as `LONDON_TIMEZONE` which is referencing the IANA timezone name, which is static so shouldn't be an env var. If we're after `DEFAULT_TIMEZONE` then I think we should look to refactor the variables, which is outside this scope (and will need to be considered whereever we transform timezones -- ie if we changes the `TZ` var to Europe/Paris, would we want to change _all_ cases of where we show time for this? Would we want to save things in the database like this etc?

Basically -- I know you'll have a great reason for this -- but for now as this is such a big ticket etc, and the changes could introduce subtle bugs if not considered, I might leave it if okay.

Reply:
If we want to use env vars instead of importing a constant, I suggest we do this in a different PR.

This PR is purely aimed at creating a DRY/common instance of `Europe/London`, some of which were hard coded.

Also btw @AlexBramhill maybe should rename the PR title to be `chore` instead of `feature`

Reply:
updated title

Reply:
It is a minor change with values always set to `Europe/London` reason I suggest using `TZ` is to ensure all the Timezone values are referred from a single point during microservice initialisation locally, CICD or on Azure.

@ttbarnes understand this PR aims to make it DRY what I am suggesting is within those remits by not repeating a value which is already specified in env, existing PR or a new PR is both acceptable as long as it is updated since time is being invested in keeping the code DRY.

Reply:
Thanks @abhi-markan makes sense to me, I think we should do it as part of this PR whilst we're here. Unless it's blocking SSO in which case i'd suggest a new PR.

I understand Alex's concern about changing the TZ var to another timezone, but this seems like an extremely rare potential future requirement to me. **UK**EF

Reply:
Could I ask why we need to use an env var for it please?

Reply:
I'm not sure I agree this should be an env var. We do not have checking across services that these env vars are all identical, and its important to ensure they are all consistently set. Further, we have no way to verify that all our env vars are set to a valid IANA time zone. 

Regardless:
- I've updated this pattern to use a config, as it is no longer a constant.
- I've added validation that the env var is a string (so it'll fail if missing). 
- I've renamed the timezone to `DEFAULT` to reflect that this is the `DEFAULT` timezone so it won't be incorrectly used by people looking for `LONDON` explicitly (If we ever have to use the `TZ` env var, we will most likely need to reference LONDON timezone in the future, considering UKEF is a UK business)
- I've not added a default to this env var, as adding such a fallback would allow services to easily get out of sync accidentally
- I've had a brief skim of the code, and I don't think this will have an effect on existing code, but timezone changes can be prone to error.

Reply:
I would like to pull the above change, and keep it static, as it adds additional complexity for the release of SSO which is out of scope (as seen in all tests failing due to env vars not being configured properly). Mind if I revert @abhi-markan